features/index_page.feature

```diff
@@ -13,3 +13,8 @@ Feature: index page with a list restaurants
     Given I am on the index page
     And I should see "List Of Restaurants"
     And I should see "NisseKebbab"
+
+  @javascript
+  Scenario: Loading the map on the index page
+    Given I am on the index page
+    And the map has been loaded
```

Comment:
What you are writing now, its expected that the scenario continues. 
Use Then if its the last step and also the text Thomas mentioned makes more sense.